edith-step/src/components/FileUploadModalBox.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import Modal from 'react-bootstrap/Modal';
+import PropTypes from 'prop-types';
+
+/**
+ * Modal Box where user can select a file to upload.
+ */
+class FileUploadModalBox extends React.Component {
+  /**
```

Comment:
Is there supposed to be a space between the class declaration and the comment?

Reply:
hmm, i guess the general form is to be supposed to be that way ?
https://google.github.io/styleguide/jsguide.html#jsdoc-general-form